src/components/MapView/MapView.tsx

```diff
@@ -190,6 +237,22 @@ const MapView = forwardRef<MapViewHandle, ComponentProps>(
 
                     {directionCoordinates && <Direction coordinates={directionCoordinates} />}
                 </Mapbox.MapView>
+                <View style={[styles.pAbsolute, styles.p5, styles.t0, styles.r0, {zIndex: 1}]}>
+                    <PressableWithoutFeedback
```

Comment:
Coming from https://github.com/Expensify/App/issues/43058. We missed adding hover and press effect on this button.

Reply:
We might have missed this but the @Expensify/design explicitly stated several times that we wanted this button behavior to match all of our other default buttons. Don't our default buttons all have a hover and press style?